packages/interaction/src/interactiveTarget.ts

```diff
@@ -1,3 +1,56 @@
+import type { InteractionTrackingData } from './InteractionTrackingData';
+
+export type Cursor = 'auto'
+    | 'default'
+    | 'none'
+    | 'context-menu'
+    | 'help'
+    | 'pointer'
+    | 'progress'
+    | 'wait'
+    | 'cell'
+    | 'crosshair'
+    | 'text'
+    | 'vertical-text'
+    | 'alias'
+    | 'copy'
+    | 'move'
+    | 'no-drop'
+    | 'not-allowed'
+    | 'e-resize'
+    | 'n-resize'
+    | 'ne-resize'
+    | 'nw-resize'
+    | 's-resize'
+    | 'se-resize'
+    | 'sw-resize'
+    | 'w-resize'
+    | 'ns-resize'
+    | 'ew-resize'
+    | 'nesw-resize'
+    | 'col-resize'
+    | 'nwse-resize'
+    | 'row-resize'
+    | 'all-scroll'
+    | 'zoom-in'
+    | 'zoom-out'
+    | 'grab'
+    | 'grabbing';
+
+export interface IHitArea {
+    contains(x: number, y: number): boolean;
+}
+
+export interface InteractiveTarget {
+    interactive: boolean;
+    interactiveChildren: boolean;
+    hitArea: IHitArea;
+    cursor: Cursor;
```

Comment:
Cursor can actually be any string - it is a dictionary key or any valid CSS value for `cursor` (including the url() stuff). I really need to set up an example of all the capabilities.